labelparse.py

```diff
@@ -37,19 +33,19 @@ def repo_fetch():
 
     for repo in dtk_repo_names:
         response = requests.get(
-            'https://api.github.com/repos/newrelic/' + str(repo) + '/issues')
-        if response.status_code == 200:
+            f'https://api.github.com/repos/newrelic/{repo}/issues'
+        )
+        if response:
             for issue in response.json():
                 print_issue(issue)
-                for label in issue['labels']:
-                    if label['name'] in labels_count:
-                        labels_count[label['name']] += 1
-                    else:
-                        labels_count[label['name']] = 1
+                for label in issue.get('labels', []):
```

Comment:
In `print_labels()`, you checked to make sure `labels` was a key. Here, you didn't. So I modified this a little to:

1. Protect from that same error case
2. Go ahead and return an empty list from `issue.get()` if there is no key so that the `for` loop works (it would just not execute any iterations with an empty list)